css/styles.css

```diff
@@ -659,3 +659,286 @@ p#modal-close {
 .modal-btn {
   margin-top: 2rem;
 }
+
+/* Desktop Version */
+
+@media screen and (min-width: 768px) {
+  .headline-section {
+    height: 100vh;
+    background: var(--white) url(../images/background.svg);
+    object-fit: cover;
+    background-size: 100%;
+    background-repeat: no-repeat;
+    border-bottom-left-radius: 75px;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    padding-left: 10rem;
+  }
+
+  .nav-menu li a {
+    color: #000;
+    list-style-type: none;
+    font-weight: 450;
+    font-size: 2.1rem;
+  }
+
+  .nav-bar .fa-bars {
+    display: none;
+  }
+
+  ul.navigation-bar.nav-menu {
+    width: 0;
+    height: 0;
+    background: none;
+    display: flex;
+    margin: 0.1rem 60rem;
+    padding-bottom: 3rem;
+    position: fixed;
+  }
+
+  .nav-bar {
+    display: inline-block;
+  }
+
+  /* Contact Section */
+
+  .contact-section {
+    height: 120vh;
+    background: #6070ff url(../images/contactform.svg);
+    object-fit: cover;
+    background-size: 100%;
+    background-repeat: no-repeat;
+    border-top-left-radius: 75px;
+    color: var(--white);
+    width: 100%;
+    margin-top: 0;
+    text-align: center;
+  }
+
+  .contact-btn {
+    margin-top: 1.2rem;
+  }
+
+  .contact-container {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+    margin-bottom: 6rem;
```

Comment:
- [ ] Please expand a little bit the content in the "contact" section to follow the Figma design.

### Your design
![Screenshot from 2023-03-31 15-07-02](https://user-images.githubusercontent.com/91142432/229166321-017994c6-4ce1-41ce-ae9e-87a2e608d0e6.png)

### What it supposes to look like
![Screenshot from 2023-03-31 15-39-50](https://user-images.githubusercontent.com/91142432/229166676-a115d45c-dc98-489a-8616-e3ca8dea735f.png)